sc/src/main/java/uk/ac/ebi/atlas/search/GeneSearchServiceDao.java

```diff
@@ -115,10 +115,9 @@ public GeneSearchServiceDao(NamedParameterJdbcTemplate namedParameterJdbcTemplat
                         subFacetValues -> subFacetValues.get("val").toString(),
                         subFacetValues -> subFacetFields
                                 .stream()
-                                .map(SingleCellAnalyticsCollectionProxy.SingleCellAnalyticsSchemaField::name)
                                 .collect(Collectors.toMap(
-                                        subFacetField -> subFacetField,
-                                        subFacetField -> getValuesForFacetField(subFacetValues, subFacetField)
+                                        SingleCellAnalyticsCollectionProxy.SingleCellAnalyticsSchemaField::displayName,
```

Comment:
In these cases, in order to improve readability, I tend to add manually:
```import <package>.SingleCellAnalyticsCollectionProxy.SingleCellAnalyticsSchemaField```

Reply:
Done